tests/src/readytorun/main.cs

```diff
@@ -420,6 +463,7 @@ static void GenericLdtokenFieldsTest()
 
     static void RunAllTests()
     {
+        TestCtorAttr();
```

Comment:
I don't think that whoever owns the ready to run test would approve of random stuff showing up in here. This is not related to the R2R feature. @weshaggard Do you know if there's any sort of process for checking in regression tests like this? I assume it would have to be something like a disabled corefx test until new System.Private.CoreLib propagates to corefx.